### PR TITLE
MON-669: Add etcd ServiceMonitor, before that was managed by cluster-…

### PR DESCRIPTION
**Pull request by machine424**

### CODE REVIEW

File: bindata/etcd/minimal-sm.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: etcd
+  namespace: openshift-etcd-operator
```
Comment (machine424) [inline bindata/etcd/minimal-sm.yaml:5]: don't know if it's ok to deploy resources on operator namespace
(still think this makes more sense on `openshift-etcd` ns, but your call)

Comment (tjungblu) [reply bindata/etcd/minimal-sm.yaml:5]: absolutely fine

File: bindata/etcd/minimal-sm.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        keySecret:
+          name: etcd-metric-client
+          key: tls.key
+  jobLabel: k8s-app
+  namespaceSelector:
+    matchNames:
+    - openshift-etcd
+  selector:
+    matchLabels:
+      k8s-app: etcd
```
Comment (tjungblu) [inline bindata/etcd/minimal-sm.yaml:38]: super nitpick: newline

Comment (tjungblu) [reply bindata/etcd/minimal-sm.yaml:38]: same on sm.yaml below

**Comment by hasbro17** [comment]
/override ci/prow/e2e-aws-ovn-etcd-scaling
Seems to be passing other than the plaguing disruption tests flakes.

LGTM whenever the nits from @tjungblu are addressed.

File: bindata/etcd/minimal-sm.yaml (hunk 2 above)
Comment (machine424) [reply bindata/etcd/minimal-sm.yaml]: Done

**Pull request merged**